<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCB</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/qrc/logo_app.ico</normaloff>:/qrc/logo_app.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-image: url(:/qrc/background_400x300.png);
    background-repeat: no-repeat;
}
</string>
  </property>
  <widget class="QRadioButton" name="radioButton_uart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 16px &quot;Arial&quot;;border: 2px dashed black;</string>
   </property>
   <property name="text">
    <string>UART</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 16px &quot;Arial&quot;;border: 2px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Communication</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ip">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 16px &quot;Arial&quot;;border: 2px dashed black;</string>
   </property>
   <property name="text">
    <string>TCP/UDP</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_bl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 16px &quot;Arial&quot;;border: 2px dashed black;</string>
   </property>
   <property name="text">
    <string>BlueTooth</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
